frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -612,8 +667,16 @@ function EditAnsibleJobModal(props: {
                     }}
                     isRequired
                 >
-                    {props.ansibleTowerTemplateList
-                        ? props.ansibleTowerTemplateList?.map((name) => (
+                    {filterForJobTemplates
+                        ? props.ansibleTowerTemplateList
+                            ? props.ansibleTowerTemplateList?.map((name) => (
+                                  <SelectOption key={name} value={name}>
+                                      {name}
+                                  </SelectOption>
+                              ))
+                            : undefined
```

Comment:
Sonar is complaining about the nested ternaries here, and I agree.

How about doing something like this early in the render function:
```
const { ansibleTowerTemplateList = [], ansibleTowerWorkflowTemplateList = [] } = props
```